### ISSUE TITLE
Question regarding Training Signal Sources

### ISSUE THREAD

**Issue by LeoXinhaoLee** [gaozhangyang/SimVP-Simpler-yet-Better-Video-Prediction#10]
Hi, thank you very much for releasing code for this inspiring work. Regarding the prediction length, say we input 10 previous frames and output 10 future frames, if our final goal is to predict 1 future frame at **evaluation time**, do we also calculate and backpropagate the loss on the 2nd to the 9th future frames or just on the 1st future frame **during training**?

Thank you very much for your time and help!

**Comment by gaozhangyang** [comment]
We will ignore the the loss on the 2nd to the 9th future frames. We just impose loss on the 1st future frame during training
